servers/Azure.Mcp.Server/TROUBLESHOOTING.md

```diff
@@ -53,7 +95,7 @@ Use the `--namespace` option to expose only tools for specific Azure services:
   "servers": {
     "Azure Storage": {
       "type": "stdio",
-      "command": "<absolute-path-to>/azure-mcp/core/src/AzureMcp.Cli/bin/Debug/net9.0/azmcp[.exe]",
+      "command": "<absolute-path-to>/mcp/servers/Azure.Mcp.Server/src/Debug/net10.0/azmcp[.exe]",
```

Comment:
What are these examples for? Do we need to really hard-code the full path?

Reply:
These are examples of using a local build of Azure MCP in an `mcp.json`. Ex, I have this locally for testing end-to-end:

```json
{
	"servers": {
		"azure-mcp-local-debug": {
			"type": "stdio",
			"command": "npx",
			"args": [
				"-y",
				"D:\\GitHub\\mcp\\servers\\Azure.Mcp.Server\\src\\bin\\Debug\\net9.0\\azmcp.exe",
				"server",
				"start",
				"--mode",
				"namespace"
			]
		}
	}
}
```